python/ray/data/impl/arrow_block.py

```diff
@@ -141,11 +143,11 @@ def builder() -> ArrowBlockBuilder[T]:
     def _empty_table() -> "pyarrow.Table":
         return ArrowBlockBuilder._empty_table()
 
-    def _sample(self, n_samples: int, key: SortKeyT) -> "pyarrow.Table":
+    def _sample(self, n_samples: int, key: "SortKeyT") -> "pyarrow.Table":
```

Comment:
I think we might be able to get rid of SortKeyT as well by just using KeyFn

Reply:
It seems to be a little different due to the inclusion of `Tuple[field, desc]`. I think it's ok to leave for now since SortKeyT is not exposed in the public API.